catalog/nsili/catalog-nsili-source/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -21,23 +21,27 @@
            http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.1.0.xsd">
 
     <!-- Metacard Type Subclasses to register -->
-    <bean id="nsiliMetacardType" class="org.codice.alliance.nsili.common.NsiliMetacardType"/>
-    <bean id="nsiliCxpMetacardType" class="org.codice.alliance.nsili.common.NsiliCxpMetacardType"/>
-    <bean id="nsiliGmtiMetacardType"
-          class="org.codice.alliance.nsili.common.NsiliGmtiMetacardType"/>
-    <bean id="nsiliImageryMetacardType"
-          class="org.codice.alliance.nsili.common.NsiliImageryMetacardType"/>
-    <bean id="nsiliIRMetacardType" class="org.codice.alliance.nsili.common.NsiliIRMetacardType"/>
-    <bean id="nsiliMessageMetacardType"
-          class="org.codice.alliance.nsili.common.NsiliMessageMetacardType"/>
-    <bean id="nsiliReportMetacardType"
-          class="org.codice.alliance.nsili.common.NsiliReportMetacardType"/>
-    <bean id="nsiliRfiMetacardType" class="org.codice.alliance.nsili.common.NsiliRfiMetacardType"/>
-    <bean id="nsiliTaskMetacardType"
-          class="org.codice.alliance.nsili.common.NsiliTaskMetacardType"/>
-    <bean id="nsiliTdlMetacardType" class="org.codice.alliance.nsili.common.NsiliTdlMetacardType"/>
-    <bean id="nsiliVideoMetacardType"
-          class="org.codice.alliance.nsili.common.NsiliVideoMetacardType"/>
+    <bean id="nsiliMetacardType" class="ddf.catalog.data.impl.MetacardTypeImpl">
+        <argument value="nsili.aggregate"/>
+        <argument>
+            <list>
+                <bean class="ddf.catalog.data.impl.types.AssociationsAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.ContactAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.MediaAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.DateTimeAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.LocationAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.ValidationAttributes"/>
+                <bean class="org.codice.alliance.catalog.core.api.impl.types.IsrAttributes"/>
+                <bean class="org.codice.alliance.catalog.core.api.impl.types.SecurityAttributes"/>
```

Comment:
I see `Associations.RELATED` being set elsewhere, but not being added to the NSILI Metacard Type